src/multivariate/multinomial.jl

```diff
@@ -23,18 +23,21 @@ struct Multinomial{T<:Real} <: DiscreteMultivariateDistribution
     n::Int
     p::Vector{T}
 
-    function Multinomial{T}(n::Integer, p::Vector{T}) where T
+    Multinomial{T}(n::Integer, p::Vector{T}) where {T} = new{T}(round(Int, n), p)
+end
+
+function Multinomial(n::Integer, p::Vector{T}; arg_check = true) where {T<:Real}
```

Comment:
Pure bikeshedding, but I'd have called it `checkargs` or `check_args`. Also I don't think we use spaces around `=` for keyword arguments.

Reply:
I was thinking of `check_args` as the operation / action, while `arg_check` stands for "argument check", as in "is there an argument check"

Reply:
Yeah, but I have the impression that we generallly use verbs rather that nouns.